dali/test/python/test_video_pipeline.py

```diff
@@ -47,11 +46,12 @@
 
 
 class VideoPipe(Pipeline):
-    def __init__(self, batch_size, data, shuffle=False, stride=1, step=-1, device_id=0, num_shards=1,
-                 dtype=types.FLOAT, sequence_length=COUNT):
-        super(VideoPipe, self).__init__(batch_size, num_threads=2, device_id=device_id, seed=12)
-        self.input = ops.readers.Video(device="gpu", filenames=data, sequence_length=sequence_length,
-                                       shard_id=0, num_shards=num_shards, random_shuffle=shuffle,
+    def __init__(self, batch_size, data, shuffle=False, stride=1, step=-1, device_id=0,
+                 num_shards=1, dtype=types.FLOAT, sequence_length=COUNT):
+        super().__init__(batch_size, num_threads=2, device_id=device_id, seed=12)
+        self.input = ops.readers.Video(device="gpu", filenames=data,
+                                       sequence_length=sequence_length, shard_id=0,
+                                       num_shards=num_shards, random_shuffle=shuffle,
```

Comment:
```suggestion
                                       sequence_length=sequence_length,
                                       shard_id=0, num_shards=num_shards,
                                       random_shuffle=shuffle,
```
...or just go with 1 argument per line

Reply:
done